### DISCUSSION TITLE
🐛 Mouse scroll not working

### DISCUSSION THREAD

**Discussion by dougg0k** [dandavison/delta#1626] [General]
Hi, 

I have been trying to get delta to work with mouse scrolls, but so far, it's not working.

I tried setting the DELTA_PAGER var mentioned at https://dandavison.github.io/delta/tips-and-tricks/mouse-scrolling.html but didnt work.

I tried working with `bat` also doesnt work.

Mouse scroll straight using `less` or `bat` in a file, do work.

I had another issue open because of this in another repo till I found that delta were the problem. https://github.com/sharkdp/bat/issues/2860

Less is a bit outdated here, but since it works the functionality works straight in a file, it also should on delta. 
```
nu ❯ less --version
less 590 (GNU regular expressions)
Copyright (C) 1984-2021  Mark Nudelman
```

---

Accordingly to this https://superuser.com/a/1453863 it should support mouse scroll from 530 forward.

**Comment by dandavison** [comment]
Hi @dougg0k, can you check the existing issues on this topic? E.g.
https://github.com/dandavison/delta/issues/630
https://github.com/dandavison/delta/issues/58

I don't use mouse-scrolling in less/delta I'm afraid and I don't always find it easy to know how to reproduce the problems people report with it.

> Less is a bit outdated here, but since it works the functionality works straight in a file, it also should on delta.

Note that it's possible that delta could be picking up a different version of less, so you could try setting the absolute path to less explicitly in delta's options or an env var.

**Reply by dougg0k** [reply]
It's been resolved already https://github.com/sharkdp/bat/issues/2860#issuecomment-1942060236

**Comment by Morgan222453** [comment]
gj rer
